source/includes/fundamentals/code-snippets/indexes/atlasVectorSearch.go

```diff
@@ -0,0 +1,161 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"go.mongodb.org/mongo-driver/v2/bson"
+	"go.mongodb.org/mongo-driver/v2/mongo"
+	"go.mongodb.org/mongo-driver/v2/mongo/options"
+)
+
+func main() {
+	ctx := context.Background()
+
+	// Retrieves your Atlas connection string
+	uri := os.Getenv("MONGODB_ATLAS_URI")
+	if uri == "" {
+		log.Fatal("MONGODB_ATLAS_URI environment variable is not set")
+	}
+
+	// Connect to your Atlas cluster
+	clientOptions := options.Client().ApplyURI(uri)
+	client, err := mongo.Connect(ctx, clientOptions)
+	if err != nil {
+		log.Fatalf("Failed to connect to the server: %v", err)
+	}
+	defer func() {
+		if err  := client.Disconnect(ctx); err != nil {
+			log.Fatalf("Failed to disconnect: %v", err)
+		}
+   }()
```

Comment:
```suggestion
  	 }()
```
My bad, I think my code suggestion didn't indent this correctly.